README.md

```diff
@@ -56,9 +56,44 @@
 
 [![Product Name Screen Shot][product-screenshot]](https://example.com)
 
-Robusta's Kubernetes Resource Recommender (KRR) is a tool that helps users to optimize the resource usage of their Kubernetes clusters. It is based on the Prometheus monitoring system and the Kubernetes API. It is designed to be used both as a CLI tool or as a Kubernetes operator. It is also designed to be easily integrated into Robusta UI, so if you are already using it you can easily start using KRR.
+Robusta KRR (Kubernetes Resource Recommender) is an extensible CLI tool designed to help optimize resource allocation in Kubernetes clusters. It gathers pod usage data from Prometheus, and applies customizable strategies to calculate CPU and memory requests and limits recommendations. This helps you get the most out of your cluster resources and improve overall efficiency.
 
-This tool is also designed to be easily extensible. It is possible to add additional calculation strategies yourself, if you want to use a different strategy than the ones provided by default.
+### Features
```

Comment:
I would emphasize that it runs out of cluster

Reply:
We're also emphasizing too much IMO the extensibility. I would keep the part on `Extensible Strategies` but make it last in the list and remove Custom Formatters. The vast majority of users wont extend, which is fine. Emphasize the builtin stuff which tbh is excellent!